src/utils.ts

```diff
@@ -93,7 +93,11 @@ const checkOk = async (response: Response): Promise<void> => {
  */
 function getPlatform(): string {
   if (typeof window !== 'undefined' && window.navigator) {
-    return `${window.navigator.platform.toLowerCase()} Browser/${navigator.userAgent};`
+    const platform = window.navigator.platform || window.navigator.userAgentData?.platform;
+    if (!platform) {
+      return '';
+    }
```

Comment:
```suggestion
const platform = 
      window.navigator.platform || 
      window.navigator.userAgentData?.platform ||
      'unknown';
```

I think we can simplify to this

Reply:
It means we'd return `unknown Browser/${navigator.userAgent};`.
If that's ok, I'm ok 😄